src/types/Events.ts

```diff
@@ -132,6 +136,7 @@ export class GeneralMeeting extends SHPEEvent {
         this.locationName = null;
         this.geolocation = null;
         this.general = true;
+        this.notificationSent = true;
```

Comment:
Is there any reason why `this.notificationSent = true;` is in every class? These classes are mainly used for even creation, so shouldn't this be set to false initially?

Also, If every class has the same default value, we can just set the value in the parent class.